.editorconfig

```diff
@@ -12,4 +12,4 @@ end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
 indent_style = space
-indent_size = 4
+indent_size = 2
```

Comment:
PSR-2, is the PHP community accepted coding style standard, consider adhering to it,

https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md

- Code MUST use 4 spaces for indenting, not tabs.